packages/vector_graphics/lib/vector_graphics.dart

```diff
@@ -199,6 +200,12 @@ class _VectorGraphicWidgetState extends State<VectorGraphic> {
         height = width / pictureInfo.size.width * pictureInfo.size.height;
       }
 
+      double scale = 1.0;
+      if (width != null && height != null) {
```

Comment:
By now the width and height are not null. The compiler can't figure this out, but it's definitely true.

Adding an assert to that effect should be fine here.

I'm really wondering if we shouldn't just be scaling the width and the height here rather than adding a property on the RO for this though.

Reply:
Oh, we can't because we need to scale the canvas by this before rasterizing. This is something like applying a DPR. I think I get it now. But we can still assert here rather than doing an if.

Reply:
you got it